src/components/Charts/Charts.tsx

```diff
@@ -1,4 +1,95 @@
-export default function Charts() {
-  // RENDER
-  return <div className="Charts">Charts</div>;
+// Libs
+import * as React from "react";
+import { Container } from "@material-ui/core";
+import {
+	ResponsiveContainer,
+	CartesianGrid,
+	XAxis,
+	YAxis,
+	LineChart,
+	Line,
+	Tooltip,
+	Legend,
+} from "recharts";
+
+// Assets
+import { configMonths, configGroups, iTransaction } from "../../types";
+
+export default function Charts({
+	cleanTransactions,
+}: {
+	cleanTransactions: Array<iTransaction>;
+}) {
+	// Definitions
+	const [chartData, setChartData] = React.useState<Array<{}>>([{}]);
```

Comment:
Typescript n'arrive pas à détecter le type lui-même?

Reply:
Removed in later PR.